README.md

```diff
@@ -1105,10 +1105,20 @@ Be advised that redefining an association is an undocumented feature of Rails.
 ### 5.c. Generators
 
 PaperTrail has one generator, `paper_trail:install`. It writes, but does not
-run, a migration file.
-The migration adds (at least) the `versions` table. The
-most up-to-date documentation for this generator can be found by running `rails
-generate paper_trail:install --help`, but a copy is included here for
+run, a migration file. The migration creates the `versions` table.
+
+```
+rails generate paper_trail:install --with-changes
+```
+
+The `--with-changes` option is recommended. It adds an `object_changes` column,
+which enables [diffing](#3c-diffing-versions) and [queries](#3e-queries). This
+optional column can be deleted later, if necessary.
```

Comment:
Do we need to discuss or make clear any limitations for MySQL users for the json data type? 

I *did* notice in my last few PRs that when we are testing the JSON `object_changes_*` methods, we only, I believe, do so only in Postgres, not MySQL, so I don't know if we have a full picture of what the MySQL compatibility story is here - unless I was misreading the test suite setup.

Reply:
> I _did_ notice in my last few PRs that when we are testing the JSON `object_changes_*` methods, we only, I believe, do so only in Postgres ..

That's correct. I'll look into enabling those tests for MySQL.